VOTable.tex

```diff
@@ -679,10 +679,27 @@ \subsection{\elem{COOSYS} Element}
 be referred to via the \attr{ref} attribute of the position components,
 which is generally the case.
 
-\item[\attr{system}] Specifies the coordinate system among
-\verb|"ICRS"|, \verb|"eq_FK5"|,
-\verb|"eq_FK4"|, \verb|"ecl_FK4"|, \verb|"ecl_FK5"|, \verb|"galactic"|,
-\verb|"supergalactic"|.
+\item[\attr{system}] Specifies the reference frame of the coordinates.
+The value must be taken from the IVOA \emph{refframe} vocabulary
+(\url{http://www.ivoa.net/rdf/refframe}).  At the time of writing, this
+vocabulary includes the terms:
+% GENERATED: !vocterms refframe
+\textsl{AZ\_EL},
+\textsl{BODY},
+\textsl{ecl\_FK4},
+\textsl{ECLIPTIC},
+\textsl{EQUATORIAL},
+\textsl{FK4},
+\textsl{FK5},
+\textsl{GALACTIC},
+\textsl{GALACTIC\_I},
+\textsl{GENERIC\_GALACTIC},
+\textsl{ICRS},
+\textsl{SUPER\_GALACTIC},
```

Comment:
With this change to refer to the vocabulary document, some of the existing values have changed their case and here added the `_`.  While it does say below that clients should handle unknown frames gracefully, for backward compatibility should we give a specific recommendation on how to handle the listed values from VOTable 1.4?  Perhaps that those values should continue to be supported but should be considered deprecated?  We could recommend version-specific parsing, but I wonder if being a little more flexible will help some with interoperability.